src/components/checkbox/index.ts

```diff
@@ -0,0 +1,5 @@
+import Checkbox from './Checkbox';
+
+export * from './Checkbox';
```

Comment:
this is not quite right.
We should export values from `Checkbox.types`, `Checkbox.constants` and `Checkbox.props` here.
`./Checkbox` does not have any member exports.